pkg/kubeclient/toolchainstatus.go

```diff
@@ -6,6 +6,8 @@ import (
 	crtapi "github.com/codeready-toolchain/api/api/v1alpha1"
 )
 
+const ToolchainStatusName = "toolchain-status"
+
 const (
 	toolchainStatusResourcePlural = "toolchainstatuses"
 )
```

Comment:
this can be removed & replaced by the `ToolchainStatusPlural` constant from the new `resources.go`

Reply:
good catch! done in 67e4d1e7f601312db32f160ea3ba93cc35250a64